scripts/preprocessing/04a-run_ica.py

```diff
@@ -300,14 +310,10 @@ def run_ica(subject, session=None):
         epochs = make_epochs(
             raw=raw,
             event_id=event_id,
-            tmin=config.epochs_tmin,
-            tmax=config.epochs_tmax,
-            metadata_tmin=config.epochs_metadata_tmin,
-            metadata_tmax=config.epochs_metadata_tmax,
-            metadata_keep_first=config.epochs_metadata_keep_first,
-            metadata_keep_last=config.epochs_metadata_keep_last,
```

Comment:
you removed this on purpose?

Reply:
Yes, we only need metadata when creating the non-ICA epochs (step 3)